### ISSUE TITLE
any plans for more development?

### ISSUE THREAD

**Issue by Festeazy** [berrnd/locory#1]

**Comment by berrnd** [comment]
Most probably not. I personally still run and use this to track my location history, but will most likely migrate/switch to use OwnTracks (and tooling around that) anytime soon™.

**Comment by Festeazy** [comment]
would you be able to docker-ize this?

**Comment by berrnd** [comment]
No, I personally don't use Docker for anything, too much restrictions for me most of the time.

**Comment by userbradley** [comment]
Shame, would be cool to feed it `.gpx` files! 

@Festeazy if you can get a list of PHP dependencies for this, I can try work on a container for you?

**Comment by berrnd** [comment]
So much things are missing here and OwnTracks ([link1](https://owntracks.org/booklet/), [link2](https://github.com/owntracks/frontend)) already has them, so I just see the point to go on on this.

This here is currently basically nothing. Coordinates in a database + a date range selector to display the corresponding points on a map.

My biggest personal wish for such a thing is map matching ([link](https://github.com/graphhopper/graphhopper#map-matching)), which means to re-map tracked coordinates to a "real route" (so for example to match them on a street). I track them via Tasker in a very low power way, due to that the accuracy is not so high for most tracked points.

But it's so much easier und makes so much more sense to not maintain everything from the ground up yourself und start building on an existing (good) solution, when there is one. That's the way to go for me personally, I just don't have the time (and desire) to do it the other way. But hey, this is Open Source, everyone can take the project and bring it to live again.

I haven't digged in that deep if map matching is already possible somehow with OwnTracks. If I have migrated to it and that's not the case, I will probably build something around that.

**Comment by Dtrieb** [comment]
On Android you can also give my app a try:

[https://play.google.com/store/apps/details?id=net.schlapa.saywhereiam](https://play.google.com/store/apps/details?id=net.schlapa.saywhereiam)
